data/items/items.xml

```diff
@@ -2936,7 +2937,7 @@
 		<attribute key="description" value="You see the engraving of a white raven on its surface." />
 		<attribute key="weight" value="420" />
 	</item>
-	<item id="2328" article="a" name="egg">
+	<item id="2328" article="an" name="egg" editorsuffix=" (Old Phoenix egg)">
```

Comment:
Are you sure this is correct? From what I saw he is this item there: (ClientID 3606) ![image](https://user-images.githubusercontent.com/8551443/67147602-90214500-f26c-11e9-9d8c-03805dae8554.png)
The Phoenix is this (ClientID 3215): 
![image](https://user-images.githubusercontent.com/8551443/67147629-c959b500-f26c-11e9-9778-1e0c380b6163.png)